app.js

```diff
@@ -0,0 +1,17 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+const ul = document.createElement('ul');
+const li = document.createElement('li');
+
+addTask = (e) => {
+    e.preventDefault();
```

Comment:
You can just use listener on input, so you don't have to prevent default action in form.